docs/topics/i18n/translation.txt

```diff
@@ -1102,6 +1104,14 @@ directories listed in :setting:`LOCALE_PATHS` have the highest precedence with
 the ones appearing first having higher precedence than the ones appearing
 later.
 
+.. note::
```

Comment:
Use `.. admonition:: <descriptive title>` instead of ".. note::" to improve skimmability.